resources/submission.scss

```diff
@@ -54,6 +54,10 @@
         .score {
             font-size: 1.3em;
             color: #000;
+
+            .grading {
```

Comment:
Can we call this something more descriptive, like `grading-spinner`?

Reply:
Done